recipes/lerclib/all/patches/add-CMakeLists-and-static.patch

```diff
@@ -0,0 +1,61 @@
+--- /dev/null
++++ b/CMakeLists.txt
+@@ -0,0 +1,24 @@
++cmake_minimum_required(VERSION 3.0.2)
++project(Lerc)
++
++file(GLOB SOURCES 
++  "src/LercLib/*"
++  "src/LercLib/Lerc1Decode/*"
++)
++
++add_library(LercLib ${SOURCES})
```

Comment:
Usually I don't agree adding a custom cmake to the recipe, as the author provides some support. If you want to add a new file, please follow the target name used by the author (Lerc): https://github.com/Esri/lerc/blob/v2.1/build/linux/Makefile#L38

Reply:
The name in makefile is not consistent with the upcoming CMakeLists in master. I've decided to provide a CMake file through a patch, waiting for CMake support in next release.

Reply:
Indeed, I didn't check the master branch. Well, keep this name so.